tests/pyspades/test_teams.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+"""
+test pyspades/teams.pyx
+"""
+from pyspades.constants import CTF_MODE
+
+from pyspades.protocol import BaseProtocol
+
+from pyspades.server import ServerProtocol
+from twisted.trial import unittest
+
+from pyspades.team import Team
+
+
+class TestTeams(unittest.TestCase):
+    name = "Team name"
+    color = (255, 0, 255)
+
+    def test_initial_values(self):
+        team = Team(0, self.name, self.color, False, MockServerProtocol())
+        self.assertEquals(0, team.kills)
+        self.assertEquals(0, team.score)
+
+    def test_team_from_team(self):
+        team_template = Team(0, self.name, self.color, False, MockServerProtocol())
+        team_template.kills = 9
+        team_template.score = 5
+        team = Team(*team_template.get_init_values())
+        self.assertEquals(0, team.kills)
+        self.assertEquals(0, team.score)
+        self.assertEquals(team_template.name, team.name)
+        self.assertEquals(team_template.color, team.color)
+
+    def test_team_others(self):
```

Comment:
I'm thinking this test case, and the behaviour that it tests, is problematic. The other team should probably not be transferred when we make a new Team instance. We could switch this case up to make sure we don't do that.

Reply:
lin 41 and 42 are the same as lines 81 and 82 in piqueserver/game_modes/inflitration.py I wanted to check if it works the way we want to

Reply:
Fixed